package.json

```diff
@@ -0,0 +1,25 @@
+{
+  "name": "blob.js",
+  "version": "1.0.0",
```

Comment:
https://github.com/eligrey/Blob.js/blob/master/Blob.js#L3

This is a static API versioned by a date. If you need a semver use `1.0.20140724`.